packages/common/src/sites/_internal/SiteBusinessRules.ts

```diff
@@ -245,6 +246,13 @@ export const SitesPermissionPolicies: IPermissionPolicy[] = [
     dependencies: ["hub:site:workspace", "hub:site:edit"],
     environments: ["devext", "qaext", "production"],
   },
+  {
+    permission: "hub:site:workspace:assistant",
+    availability: ["alpha"],
+    licenses: ["hub-premium"],
+    dependencies: ["hub:site:workspace", "hub:site:edit"],
+    environments: ["devext", "qaext", "production"],
```

Comment:
Is there a service we could key this off of instead of listing the env's? We'd need to add an entry to `HubService` type, but that would allow us to gracefully degrade if the service is offline